CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 2.0.0 - UNRELEASED
+
+This version contains is technically a BC break because we drop PHP5 and add return 
```

Comment:
<preaching mode on>dropping PHP5 support is no BC break. it just prevents people on an obsolete stack from updating</off>

how do you mean "technically"? i think we should say: "This version is no BC break for consumers *using* HTTPlug. However, HTTP clients that *implement* HTTPlug need to adjust because we add return type declarations."